pages/index.py

```diff
@@ -265,3 +247,4 @@ def __init__(self):
             ServiceSection(),
             NewsSection(),
             ContactSection()]
+
```

Comment:
Git always complains when a file does not have a trailing newline (and rightfully so, otherwise it might have to create diffs without trailing context, so they might end up conflicting rather easily). So the blank line is somewhat intended, yet more of a result of a habit developed over the past years. If you'd like me to drop it, I'm fine with that.